components/server/src/auth/rate-limiter.ts

```diff
@@ -260,10 +261,6 @@ function getConfig(config: RateLimiterConfig): RateLimiterConfig {
             points: 10,
             durationsSec: 10,
         },
-        sendHeartBeat: {
```

Comment:
Let's keep it there, but with a much higher value as a safe-guard. Something like 100 per user, which would require you to have ~50 workspaces

Reply:
Done.

(Maybe that value should be [the default](https://github.com/gitpod-io/gitpod/blob/03ceb6dfcc8b24350133ec4a9eb2ebf0e873faa1/components/server/src/auth/rate-limiter.ts#L248-L251) in the first place :see_no_evil: )

Reply:
Yeah, probably. Let's fix in a follow-up.

Reply:
> Let's fix in a follow-up.

Issue here: https://github.com/gitpod-io/ops/issues/8812